config/kernel-monitor-counter.json

```diff
@@ -0,0 +1,26 @@
+{
+  "plugin": "custom",
+  "pluginConfig": {
+    "invoke_interval": "10m",
+    "timeout": "1m",
+    "max_output_length": 80,
+    "concurrency": 1
+  },
+  "source": "kernel-monitor",
+  "conditions": [
+    {
+      "type": "NodeRecreationRequired",
```

Comment:
This seems to be a auto repair decision. Let's keep this as a problem description.

Reply:
renamed to FrequentUnregisterNetDevice